contents/docs/privacy/index.md

```diff
@@ -19,5 +19,7 @@ In these guides, we offer advice for using PostHog in a compliant manner under t
 
 ### Further reading
 
+- [A simple guide to personal data and PII](/blog/what-is-personal-data-pii)
 - [The best HIPAA-compliant analytics tools](/blog/best-hipaa-compliant-analytics-tools/)
-- [The best GDPR-compliant analytics tools](/blog/best-gdpr-compliant-analytics-tools/)
\ No newline at end of file
+- [The best GDPR-compliant analytics tools](/blog/best-gdpr-compliant-analytics-tools/)
+- [Is Google Analytics HIPAA compliant?](/blog/is-google-analytics-hipaa-compliant)
```

Comment:
This link + the one above are not actually related to this page of docs - no user would need to go from learning about PostHog's privacy settings to then wanting to consider "best analytics tools"... they're already using us. I think this is putting marketing ahead of the docs experience, so I'd advocate for removing. If we need these kinds of internal links, dropping them between blog articles would be cleaner - good docs must have no needless fluff in, however.

The other links in here I think make total sense for a user.